src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -60,6 +60,13 @@
 
 
 @cloup.group(context_settings=CLICK_CTX_SETTINGS)
+@click.version_option(
+    None,  # version number not set, click will infer it using setuptools
+    "-v",
+    "--version",
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏`-v` is commonly used for verbosity flags. What about using `-V` (like `python`)?
```suggestion
    "-V",
    "--version",
```